### PR TITLE
chore(release): v0.13.1

### PR DESCRIPTION
**Pull request by niklasad1**

### CODE REVIEW

**Review by lexnv — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ The format is based on [Keep a Changelog].
...
+v0.13.1 is a release that fixes that documentation for feature-gated items on `docs.rs`.
+
+### [Fixed]
+- fix: generate docs for all features on docs.rs [#767](https://github.com/paritytech/jsonrpsee/pull/767)
+
+### [Changed]
+- chore(deps): update pprof requirement from 0.8 to 0.9 [#761](https://github.com/paritytech/jsonrpsee/pull/761)
+- chore(release): v0.13.0 [#764](https://github.com/paritytech/jsonrpsee/pull/764)
+
+
```
Comment (niklasad1) [inline CHANGELOG.md:20]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,17 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [v0.13.1] - 2022-05-13
+
+v0.13.1 is a release that fixes that documentation for feature-gated items on `docs.rs`.
```
Comment (TarikGul) [inline CHANGELOG.md:11]:
```suggestion
v0.13.1 is a release that fixes the documentation for feature-gated items on `docs.rs`.
```

**Review by TarikGul — APPROVED** [review]

**Review by jsdw — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,17 @@ The format is based on [Keep a Changelog].
...
+## [v0.13.1] - 2022-05-13
+
+v0.13.1 is a release that fixes the documentation for feature-gated items on `docs.rs`.
+
+### [Fixed]
+- fix: generate docs for all features on docs.rs [#767](https://github.com/paritytech/jsonrpsee/pull/767)
+
+### [Changed]
+- chore(deps): update pprof requirement from 0.8 to 0.9 [#761](https://github.com/paritytech/jsonrpsee/pull/761)
+- chore(release): v0.13.0 [#764](https://github.com/paritytech/jsonrpsee/pull/764)
```
Comment (jsdw) [inline CHANGELOG.md:18]: Nit: Do we normally mention this release PR?

Comment (niklasad1) [reply CHANGELOG.md:18]: no, let's remove it

Comment (niklasad1) [inline CHANGELOG.md:18]:
```suggestion
```

**Pull request merged**